lib/td_cache/user_cache.ex

```diff
@@ -218,24 +219,34 @@ defmodule TdCache.UserCache do
     |> Map.new()
   end
 
-  defp delete_user(id) do
-    case Redix.command!(["HMGET", "user:#{id}", "full_name", "user_name"]) do
+  defp delete_user(%{id: user_id, acl_entries: acl_entries}) do
```

Comment:
This module's public API exposes the `delete(id)` function, which is implemented by `handle_call({:delete, id}, _from, state)`. Changing this function's signature breaks this contract and impacts test cases in various services.